scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#294.sql

```diff
@@ -0,0 +1,26 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- Change releases with positive latitudes
+UPDATE animal_release SET capture_location = ST_POINT(ST_X(capture_location), ST_Y(capture_location) * (-1))
```

Comment:
Why has this data been entered into the gui incorrectly? 

Shouldn't there be some QC record to show these modifications?